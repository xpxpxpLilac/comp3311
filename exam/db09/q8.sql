drop view if exists q8;
create view q8
as
select l.country as country, count(*)
from brewers b join locations l on b.location = l.id
group by l.country
having count(*)>20
;
