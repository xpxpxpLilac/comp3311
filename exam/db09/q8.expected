Belgium|83
Canada|36
Germany|131
Switzerland|28
United Kingdom|72
United States|836
